lib/bundler/cli/outdated.rb

```diff
@@ -30,6 +30,13 @@ def run
         Bundler.definition(:gems => gems, :sources => sources)
       end
 
+      patch_level = [:major, :minor, :patch].select {|v| options.keys.include?(v.to_s) }
+      Bundler.definition.create_gem_version_promoter([]).tap do |gvp|
+        gvp.level = patch_level.first || :major
+        gvp.strict = options[:strict]
+      end
+      Bundler.load
```

Comment:
Yeah, cuz the code that's now in Bundler::CLI::Common raises an error if more than one --patch/--minor/--major option is provided, but current outdated code allows you to pass as many as you want.

Because they do different things. See [this comment](https://github.com/bundler/bundler/pull/4841#issuecomment-251122728)